arm-network/2016-09-01/swagger/network.json

```diff
@@ -4405,8 +4405,8 @@
           "description": "Health of backend server",
           "enum": [
             "Unknown",
-            "Up",
-            "Down",
+            "Healthy",
```

Comment:
We are also changing the SDK (refer azure-sdk-for-net PR) version to 8.0.0 and we will put a note to use version 8 SDK when we will announce this feature.